conans/cli/commands/new.py

```diff
@@ -14,7 +14,13 @@ def new(conan_api, parser, *args):
     """
     Create a new recipe (with conanfile.py and other files) from a predefined template
     """
-    parser.add_argument("template", help="Template name, built-in predefined one or user one")
+    parser.add_argument("template", help="Template name, built-in predefined one or user one. "
```

Comment:
I guess we are intentionally no documenting the `conan new alias ...` ?

Reply:
I didn't do it on purpose, but I think yes, it is better not document that here yet.